tests/functional/services/policy_engine/test_feed_sync.py

```diff
@@ -0,0 +1,104 @@
+import os
+
+import tests.functional.services.policy_engine.utils.api as policy_engine_api
+from tests.functional.services.policy_engine.conftest import FEEDS_DATA_PATH_PREFIX
+from tests.functional.services.utils import http_utils
+
+# TODO check schema
+
+
+class TestFeedSync:
+    @classmethod
+    def _find_by_name(cls, records, name):
+        """
+        From a list of objects/dictionaries, selects first index with matching name. Returns None if nothing is found
+        :param records: list of objects or dictionaries that are expected to have 'name' attr
+        :type records: list
+        :return: dict with matching name or 'None' if nothing found
+        :rtype: Union[dict, None]
+        """
+        for record in records:
+            if record["name"] == name:
+                return record
+        return None
+
+    @classmethod
+    def _get_vuln_ids(cls, expected_vulns):
+        """
+        From a list of expected vulns taken from feeds service, find corresponding vulnerability ids for querying anchore
+        :param expected_vulns: list of records in group from feed
+        :type expected_vulns: list
+        :return: list of vulnerabilitiy_ids as they would be stored in anchore
+        :rtype: list
+        """
+        vuln_ids = []
+        for vuln in expected_vulns:
+            if "Advisory" in vuln:
+                if "ghsaId" in vuln["Advisory"]:
+                    vuln_ids.append(vuln["Advisory"]["ghsaId"])
+                else:
+                    vuln_ids += vuln["Advisory"]["CVE"]
```

Comment:
Is there ever a case where this key will not be found, ie "ghsaId" is not in vuln["Advisory"], but neither is "CVE"

Reply:
@dspalmer99  At the moment those keys will always be found because of the static set of feed data that we are using. 

@Vijay-P What is the else condition meant to cover? Are there groups with `Advisory` that do not have ghsaId?

Reply:
There are actually no groups in the current set of static data that we are using that have Advisory but do not have ghsaId. The CVE bit was included as sort of a just-in-case. May be too permissive. Can make this tighter.

Reply:
added some comments to this code